doc/usersguide/uw2.rst

```diff
@@ -165,12 +165,12 @@ now run as if it were a standard FLUKA problem
 OpenMC-specific steps
 ~~~~~~~~~~~~~~~~~~~~~
 
-To run a OpenMC UWUW simulation, a ``dagmc.h5m`` file containing the UWUW model
-must be present in the OpenMC run directory and a ``dagmc`` element in the
-``settings.xml`` file must be set to true like so:
+To run a OpenMC UWUW simulation, a ``.h5m`` file containing the UWUW model
+must be present in the OpenMC run directory and a ``dagmc_universe`` loaded in
```

Comment:
```suggestion
must be present in the OpenMC run directory and a ``openmc.DAGMCUniverse`` loaded in
```

Should this be  a openmc.DAGMCUniverse[(https://docs.openmc.org/en/stable/pythonapi/generated/openmc.DAGMCUniverse.html#openmc-dagmcuniverse)

Reply:
I was trying to refer to this, not necessarily the underlying Python class. Would you prefer I swap out "loaded" with "used" to avoid any confusion?

```
<geometry>
  <dagmc_universe filename="file.h5m" id="1" />
</geometry>
```

Reply:
I'll suggest "present in the ``geometry.xml`` file".

Reply:
good idea, I like that wording